plugins/c8y_firmware_plugin/src/firmware_manager.rs

```diff
@@ -534,11 +521,45 @@ impl FirmwareManager {
         );
 
         if op_state == ActiveOperationState::Pending {
-            self.mqtt_client.published.send(executing_msg).await?;
+            self.mqtt_client
+                .published
+                .send(executing_msg)
+                .await
+                .map_err(MqttError::from)?;
```

Comment:
Why not just create `FirmwareManagementError` variant named `FromSendError(#[from] futures_channel::mpsc::SendError)` just like other errors, so that we can continue doing `.await?`